docs/cmd/kn_broker.md

```diff
@@ -30,5 +30,5 @@ kn broker
 * [kn broker create](kn_broker_create.md)	 - Create a broker
 * [kn broker delete](kn_broker_delete.md)	 - Delete a broker
 * [kn broker describe](kn_broker_describe.md)	 - Describe broker
-* [kn broker list](kn_broker_list.md)	 - List brokers
+* [kn broker list](kn_broker_list.md)	 - List brokers(Alias: 'ls')
```

Comment:
I wonder if "alias" (lower case) would be better/cleaner?

Reply:
Done